### ISSUE TITLE
More informative failure notification

### ISSUE THREAD

**Issue by nbgo** [dwightjack/vue-types#2]
Now it just says:
```VM224:27 [Vue warn]: Invalid prop: custom validator check failed for prop "model".```

Can validator throw with more specific reason of failure instead of simple returning true/false?

**Comment by dwightjack** [comment]
Can you post the prop shape?

**Comment by nbgo** [comment]
For example I get the message if ```id``` missed for this prop:
```model: VueTypes.shape({
			id: VueTypes.string.isRequired,
			isNew: VueTypes.bool
		}).isRequired```

**Comment by dwightjack** [comment]
Thank you. At the moment I'm not at the PC. Will check asap

**Comment by dwightjack** [comment]
Hi,

I've put up a little demo with your type definition and it seems to work: https://jsfiddle.net/dwightjack/xf2k9d6u/

If you have any further detail feel free to post it or to fork the fiddle and adapt it to your scenario.

**Comment by nbgo** [comment]
Yes, it works when validation result is success. Perhaps I was not clear enought in problem explanation. I mean the case when a validation rule is violated. Then it is desired to see specific info what is wrong instead of some common info "There is some error": https://s3.amazonaws.com/donnicky_screenshots/2016-11-14_14-22-51_52be430e-dc78-4286-83ea-caa3e157d11c.png
As I remember React PropTypes produce specific information what is wrong and it would be nice to see the same approach with Vue and your library.

**Comment by dwightjack** [comment]
Ah sorry :)

Not sure I can manage to override the validation error, but I'd be able to set a development warning with specific errors.

Thanks!
